gravityflow/gravityflow
# Add in check to ensure form ids match on entry detail page

## Description
When we load an Entry Detail/Workflow Detail page, we pass a form ID query param (`id`), but never really use it or confirm that it's the correct value for the entry being viewed. This can lead to some confusion if the ID is changed/wrong for any reason. 

This fix simply adds in a check to ensure that, if passed, the `id` param matches the retrieved Entry's `form_id` value, and if not, bails.

This resolves [gravityflow/backlog#76](https://github.com/gravityflow/backlog/issues/76).

## Testing instructions
- Create a handful of Workflow Entries from a form
- Visit the Workflow Detail page for one of them
- Ensure you see the correct entry details
- Change the `id` query param in the URL to something different and refresh
- Ensure you see an error message and that no entry details display.

## Checklist:
